tests/test_transpilers_general.py

```diff
@@ -6,6 +6,8 @@
 
 from qibolab.transpilers.general_connectivity import Transpiler
 
+np.random.seed(42)
```

Comment:
Please move this inside a specific test. Moreover now you can assert if predictions do not change between runs.

Reply:
Yesterday I had a meet with Alessandro Candido where we decided to implement more useful tests. Next week we will write a "Can_execute" function to be used both by the transpiler and tests.